packages/terra-data-grid/src/subcomponents/ColumnResizeHandle.module.scss

```diff
@@ -1,25 +1,35 @@
+/* stylelint-disable terra/custom-property-pseudo-selectors */
 // Themes
 @import '../orion-fusion-theme/ColumnResizeHandle.module';
 @import '../clinical-lowlight-theme/ColumnResizeHandle.module';
 
 :local {
   .resize-handle {
-    border-right: 2px solid transparent;
+    border-right: 4px solid transparent;
     cursor: col-resize;
     display: block;
+    height: 100%;
     position: absolute;
     right: 0;
     top: 0;
-    width: 24px;
+    width: 22px;
     z-index: 1;
 
+    &:active {
+      border-color: var(--terra-data-grid-resize-handle-active-border-color, rgba(34, 42, 46, 1)) !important;
+    }
+
     &:focus {
-      border-color: var(--terra-data-grid-resize-handle-focus-border-color, rgba(34, 42, 46, 1)) !important;
+      border-color: var(--terra-data-grid-resize-handle-hover-border-color, rgba(34, 42, 46, 0.6));
       outline: 0 !important;
     }
 
     &:hover {
       border-color: var(--terra-data-grid-resize-handle-hover-border-color, rgba(34, 42, 46, 0.6));
     }
   }
+
+  .resize-handle-selected {
```

Comment:
Should this class just be "selected" instead of "resize-handle-selected" since it is localized?  Also, we should be able to add this under the .resize-handle style block.

Reply:
Updated here: 46c86b0cfd4e7b4787ba5483f5141f63b1099093